rs/rust_canisters/random_traffic_test/src/lib.rs

```diff
@@ -81,48 +83,57 @@ impl Config {
 pub enum Reply {
     /// A response including a data payload of a distinct size was received.
     Bytes(u32),
-    /// The call was rejected synchronously by the system with an error code.
-    SynchronousRejection(i32),
-    /// The call was rejected asynchronously through a reject response, which
-    /// includes an error code and a reject message.
-    AsynchronousRejection(i32, String),
```

Comment:
It's part of the record indirectly through caps in the 'call index' and the counter. It's just that the records got absolutely spammed with the new approach of generating N calls and then awaiting them so I figured we might as well skip them since they don't really contain anything useful. But if it that turns out to be false we can always change it and I suppose write a function that skips them when printing or something. But for now I think this should work.

Reply:
Also not that the canister wasn't designed to run for a long time since the records just keep on growing. Eventually the canister will crash, but still it seems like a good idea not to have 90% synchronous rejections (at the memory limit I mean) with no more information than 'couldn't send message' in them overflowing the records.